examples/wasm.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
```

Comment:
Another nitpick: could be a typescript file instead?

Reply:
I tried to re-write in `.ts`.
I am not sure how to treat the generated `constants.js` file. It cannot be imported now, the transpiler complains about the missing declarations. It could be imported from the `ts` source, I guess. Not sure what would be the best way for a `.ts` user. Generating that file is the only way I found to access the constants from `js`.

Reply:
The solution was in front of me but I couldn't see it, There are already all the required assets for `ts` and `js` in lib. I just copied over those ones.